<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="nanonetsModelLine">
     <property name="placeholderText">
      <string>Nanonets model id giriniz</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nanonetsApiLine">
     <property name="placeholderText">
      <string>Nanonets API key giriniz</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="nanonetsKaydetBtn">
     <property name="text">
      <string>Kaydet</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
